doc/UsersGuide/source/CustomizingTheWorkflow/PhysicsSchemes.rst

```diff
@@ -0,0 +1,124 @@
+.. _PhysicsSchemes:
+
+***********************
+Physics Schemes in HAFS
+***********************
+
+This chapter contains information regarding the various physics schemes in HAFS.
+
+.. _Land/OceanSurfaceSchemes:
+
+==========================
+Land/Ocean Surface Schemes
+==========================
+
+
+Land/Ocean surface schemes provide surface temperature, heat and moisture fluxes over land, sea-ice, and ocean points. These serve as a lower boundary condition for the vertical transport in Planetary Boundary Layer (PBL) schemes.
+
+**HAFS uses:**
+
+* Noah Multi-Physics (NOAH-MP) land surface model
+* HYCOM, MOM6 for ocean models
+
+.. _SurfaceLayerScheme:
+
+====================
+Surface-Layer Scheme
+====================
+
+Surface layer schemes represent processes near the surface (<~100 m), where vertical gradients are too large to be resolved by vertical grids. They are usually based on Monin-Obukhov similarity theory.
+They provide atmospheric exchange coefficients, stability functions (surface fluxes) needed by land/ocean models and PBL.
+
+HAFS uses the GFS surface-layer model, with observation-based roughness length (particularly for strong wind conditions). Modified Zo is important for a successful simulation of TCs. Hurricane intensity is proportional to (Ck/Cd) over the ocean (Emanuel, 1995), where Cd and Ck are the sea-surface momentum and enthalpy exchange coefficients. Larger z0 leads to larger Cd and Ck.
+
+.. _PBLScheme:
+
+==========
+PBL Scheme
+==========
+
+PBL parameterization schemes handle subgrid-scale processes near the surface. In HAFS, the PBL scheme is responsible for handling turbulent scale mixing and diffusion.
+
+HAFS uses the GFS TKE-EDMF PBL scheme, with a prognostic TKE equation.
+
+Sub-grid scale turbulent flux = Flux of Large eddies + flux of local small eddies
+                                   mass-flux (MF)       eddy-diffusivity (ED)
+
+**TC intensity is sensitive to mixing/diffusion. We have two modifications:**
+
+  1. Adjustment to mixing length near the surface so that it is consistent with that used in the surface-layer scheme. (HFSA)
+  2. LES-based TC PBL adjustment including model coefficients, Richardson number calculation, high-wind mass flux, etc. (HFSB)
+
+.. _ScaleAwareConvection:
+
+===============================================
+Scale-aware Deep and Shallow Convection Schemes
+===============================================
+
+Scale-aware deep and shallow convection schemes are used when moisture convections are too small to be resolved explicitly by grid spacing; reducing the thermodynamic instability.
+HAFS uses GFS Scale-aware SAS. 
```

Comment:
```suggestion
Scale-aware deep and shallow convection schemes are used when moisture convection is too small to be resolved explicitly by grid spacing, reducing the thermodynamic instability. HAFS uses GFS Scale-Aware SAS. 
```
What is SAS? Is it Scale Aware Schemes? Or is SAS a particular scale-aware scheme?